src/Roslyn.Diagnostics.Analyzers/Core/RoslynDiagnosticsAnalyzersResources.resx

```diff
@@ -118,109 +118,109 @@
     <value>#N/A</value>
   </data>
   <data name="InvokeTheCorrectPropertyToEnsureCorrectUseSiteDiagnosticsMessage" xml:space="preserve">
-    <value>Invoke the correct property to ensure correct use site diagnostics.</value>
+    <value>Invoke the correct property to ensure correct use site diagnostics</value>
   </data>
   <data name="DoNotUseGenericCodeActionCreateToCreateCodeActionTitle" xml:space="preserve">
-    <value>Do not use generic CodeAction.Create to create CodeAction</value>
+    <value>Do not use generic 'CodeAction.Create' to create 'CodeAction'</value>
   </data>
   <data name="DoNotUseGenericCodeActionCreateToCreateCodeActionDescription" xml:space="preserve">
     <value>#N/A</value>
   </data>
   <data name="DoNotUseGenericCodeActionCreateToCreateCodeActionMessage" xml:space="preserve">
-    <value>Do not use generic CodeAction.Create to create CodeAction</value>
+    <value>Do not use generic 'CodeAction.Create' to create 'CodeAction'</value>
   </data>
   <data name="OverrideObjectEqualsObjectWhenImplementingIequatableTitle" xml:space="preserve">
-    <value>Override Object.Equals(object) when implementing Iequatable</value>
+    <value>Override 'Object.Equals(object)' when implementing 'IEquatable'</value>
   </data>
   <data name="OverrideObjectEqualsObjectWhenImplementingIequatableDescription" xml:space="preserve">
     <value>#N/A</value>
   </data>
   <data name="OverrideObjectEqualsObjectWhenImplementingIequatableMessage" xml:space="preserve">
-    <value>Override Object.Equals(object) when implementing Iequatable</value>
+    <value>Override 'Object.Equals(object)' when implementing 'IEquatable'</value>
   </data>
   <data name="DoNotInvokeDiagnosticDescriptorTitle" xml:space="preserve">
-    <value>Do not invoke Diagnostic.Descriptor</value>
+    <value>Do not invoke 'Diagnostic.Descriptor'</value>
   </data>
   <data name="DoNotInvokeDiagnosticDescriptorDescription" xml:space="preserve">
-    <value>Accessing the Descriptor property of Diagnostic in compiler layer leads to unnecessary string allocations for fields of the descriptor that are not utilized in command line compilation. Hence, you should avoid accessing the Descriptor of the compiler diagnostics here. Instead you should directly access these properties off the Diagnostic type.</value>
+    <value>Accessing the 'Descriptor' property of 'Diagnostic' in compiler layer leads to unnecessary string allocations for fields of the descriptor that are not utilized in command line compilation. Hence, you should avoid accessing the 'Descriptor' of the compiler diagnostics here. Instead you should directly access these properties off the 'Diagnostic' type.</value>
   </data>
   <data name="DoNotInvokeDiagnosticDescriptorMessage" xml:space="preserve">
     <value>Do not invoke property '{0}' on type '{1}', instead directly access the required member{2} on '{1}'</value>
```

Comment:
```suggestion
    <value>Do not invoke property '{0}' on type '{1}', instead directly access the required member '{2}' on '{1}'</value>
```

Reply:
We shouldn't change this (or at least not in this PR), the code is taking care of adding the space + wrapping in single quotes when needed: https://github.com/dotnet/roslyn-analyzers/blob/master/src/Roslyn.Diagnostics.Analyzers/Core/DoNotInvokeDiagnosticDescriptor.cs#L64

Reply:
Ah got you. Can you file a separate issue for fixing this up in a separate PR?

Reply:
#3560